README.md

```diff
@@ -95,20 +98,42 @@ This plugin decorates fastify with a `mail` object containing the following meth
 
 - `createMessage`: `function` - Generates a message from templates with context injected. 
   - Accepts the following arguments: 
-    - `recipients`: `array` - Array containing recipient[s] email address (`string`).
-    - `templates`: `string` - the relative path to the message's templates.
-    - `context`: `object` - Object containing context for the message.
+    - `message`: `array` -  Comma separated list or an array of recipients email addresses (`string`) that will appear on the To: field
```

Comment:
i think "message" here is now an object rather than an array?

Reply:
yep - spotted that and fixed already :)